src/osd/OSD.cc

```diff
@@ -6525,7 +6519,15 @@ void OSD::_dispatch(Message *m)
 
     // -- need OSDMap --
 
-  default:
+  case MSG_OSD_PG_CREATE:
+  case MSG_OSD_PG_NOTIFY:
+  case MSG_OSD_PG_QUERY:
+  case MSG_OSD_PG_LOG:
+  case MSG_OSD_PG_REMOVE:
+  case MSG_OSD_PG_INFO:
+  case MSG_OSD_PG_TRIM:
+  case MSG_OSD_BACKFILL_RESERVE:
+  case MSG_OSD_RECOVERY_RESERVE:
```

Comment:
I guess it's  better torn iao fast, we'll just have to remember to add them to this enumeration whenever we make new message types (which IIRC is why it wasn't done before). Too bad we don't have a good way to switch on the new intermediate classes.